pcp-statsd-pmda.conf

```diff
@@ -3,3 +3,6 @@ port 8125
 max_unprocessed_packets 1024
 tcp_address 0.0.0.0
 parser_type 0
+verbose 1
+debug 1
+debug_output_filename debug
```

Comment:
@natoscott is there some preferred configuration syntax for PMDAs?

Reply:
Not an actual problem, just trying to find some nice format in advance. Personally I like the traditional Linux `conf` format or `ini` style.

Reply:
BTW, there is an .ini file parser in libpcp_web - its used by pmproxy(1) and pmseries(1).  See also src/external/ini.[ch]

Reply:
Config is now ini in 38e8d4f